### PR TITLE
Make JavaScript code example actually work

### PR DESCRIPTION
**Pull request by evilpie**
`fetch` is a standard builtin. Our code examples should work.

### CODE REVIEW

File: _posts/2021-04-09-top-level-await-ships.markdown [hunk 1]
```diff
@@ -23,8 +23,8 @@ This is much simpler and robust than previous solutions, such as:
 import { process } from "./some-module.mjs";
 let output;
 async function main() {
-  const data = await fetch("./data.json");
-  const parsedData = JSON.parse(data);
+  const response = await fetch("./data.json");
+  const parsedData = JSON.parse(await response.json());
```
Comment (codehag) [inline _posts/2021-04-09-top-level-await-ships.markdown:27]: Do we nee to await twice here? i think just once above no?

Comment (evilpie) [reply _posts/2021-04-09-top-level-await-ships.markdown:27]: `json()` also returns a Promise: https://developer.mozilla.org/en-US/docs/Web/API/Body/json

Comment (codehag) [reply _posts/2021-04-09-top-level-await-ships.markdown:27]: got it, thanks for the correction. @hotsphink can you merge?

Comment (evilpie) [reply _posts/2021-04-09-top-level-await-ships.markdown:27]: Ah. I actually I just realized there is a bug. `.json()` already parses the JSON. Should we use `text()` instead?

Comment (codehag) [reply _posts/2021-04-09-top-level-await-ships.markdown:27]: sure!

Comment (codehag) [inline _posts/2021-04-09-top-level-await-ships.markdown:27]: or..?
```suggestion
  const parsedData = await response.json();
```

**Review by hotsphink — APPROVED** [review]
I think I must be stamping someone else's review? Who doesn't have privileges here that they should?

**Pull request merged**